spec/requests/feed_requests_spec.rb

```diff
@@ -46,6 +46,34 @@
       expect(xml[:updated]).to eq('2015-05-05T07:50:40+00:00')
       expect(first_entry[:published]).to eq('2015-05-05T07:50:40+00:00')
     end
+
+    it 'returns a 304 when there are no new PRs' do
+      stub_github_request
+
+      get(feed_path(owner: 'github', repo: 'code'))
```

Comment:
Maybe set a variable to `feed_path(owner: 'github', repo: 'code')` so it's clear you're hitting the same path? Like:

```ruby
path = feed_path(owner: 'github', repo: 'code')
```